modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
+
+. In the Argo CD dashboard, notice that the `cluster-configs` Argo CD application has the statuses `Missing` and `OutOfSync`. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
+
+. Click the *Sync* button on the `cluster-configs` tile, review the changes and then click the *Synchronize*.
+
+.  Notice that the `cluster-configs` Argo CD application now has the statuses `Healthy` and `Synced`. Click the `cluster-configs` tile to check the details of the synchronized resources and their status on the cluster.
+
+. Navigate to the OpenShift web console and click the *Application Launcher* and see a link to the _Red Hat Developer Blog_. 
+
+. Navigate to a and see a namespace `spring-petclinic`.
```

Comment:
I don't understand this sentence properly either, I'm afraid. Maybe these formulations belong to the guidelines for this guide and I just didn't come across it before. If so, don't worry about my comments please!

Reply:
No-no, I missed part of the sentence. You're totally right. -_-